src/store/contest/contest-api.tsx

```diff
@@ -80,3 +80,4 @@ export const {
   useFillContestStatMutation,
   useGetContestsStatQuery,
 } = contestApiSlice;
+
```

Comment:
Please change the "type any" for the responses and define a response type. For example, you can create an interface called "CreateContestResponse" and substitute it for "any."